platform/android/app/src/main/kotlin/coop/polypoly/polypod/Preferences.kt

```diff
@@ -103,5 +88,14 @@ class Preferences {
             }
             return outputMap
         }
+
+        fun setClearCorePreferences(context: Context, clear: Boolean) {
```

Comment:
Well, I did introduce `setCorePreference` now, but it would be rather inconvenient to have to implement preference clearing on every platform, hence I added an explicit clear method to Core.